src/__tests__/plugin.js

```diff
@@ -69,4 +69,25 @@ describe('gatsby-remark-embedder', () => {
       "
     `);
   });
+
+  test('logs when a transformer errors', async () => {
+    const consoleErrorMock = jest
+      .spyOn(console, 'error')
+      .mockImplementation(() => undefined);
+    const errorTransformer = {
+      getHTML: () => {
+        throw new Error('ErrorTransformer');
+      },
+      shouldTransform: () => true,
+    };
+
+    const markdownAST = getMarkdownASTForFile('ErrorTransformer', true);
+
+    await expect(
+      plugin({ cache, markdownAST }, { customTransformers: [errorTransformer] })
+    ).rejects.toMatchInlineSnapshot(`[Error: ErrorTransformer]`);
+    expect(consoleErrorMock).toHaveBeenCalledWith(
+      'The following error appeared while processing: https://error-site.com/'
+    );
```

Comment:
This (and setting spy) can now be removed, and inline snapshot on line 88 will need update now ;)